### ISSUE TITLE
[REVIEW]: pollucheck v1.0: A package to explore open-source air pollution data

### ISSUE THREAD

**Issue by whedon** [openjournals/joss-reviews#3435]
**Submitting author:** @adithirugis (<a href="http://orcid.org/0000-0002-1764-1379">Adithi Upadhya</a>)
**Repository:** <a href="https://github.com/adithirgis/pollucheck" target ="_blank">https://github.com/adithirgis/pollucheck</a>
**Version:** 1.0.0
**Editor:** @ajstewartlang
**Reviewer:** @nmstreethran, @ibarraespinosa
**Archive:** <a href="https://doi.org/10.5281/zenodo.5128607" target="_blank">10.5281/zenodo.5128607</a>

**:warning: JOSS reduced service mode :warning:**

Due to the challenges of the COVID-19 pandemic, JOSS is currently operating in a "reduced service mode". You can read more about what that means in [our blog post](https://blog.joss.theoj.org/2020/05/reopening-joss).

## Status

[![status](https://joss.theoj.org/papers/6844e6db054970a02c1ddbcf9e9f0cc2/status.svg)](https://joss.theoj.org/papers/6844e6db054970a02c1ddbcf9e9f0cc2)

Status badge code:

```
HTML: <a href="https://joss.theoj.org/papers/6844e6db054970a02c1ddbcf9e9f0cc2"><img src="https://joss.theoj.org/papers/6844e6db054970a02c1ddbcf9e9f0cc2/status.svg"></a>
Markdown: [![status](https://joss.theoj.org/papers/6844e6db054970a02c1ddbcf9e9f0cc2/status.svg)](https://joss.theoj.org/papers/6844e6db054970a02c1ddbcf9e9f0cc2)
```
**Reviewers and authors**:

Please avoid lengthy details of difficulties in the review thread. Instead, please create a new issue in the <a href="https://github.com/adithirgis/pollucheck" target="_blank">target repository</a> and link to those issues (especially acceptance-blockers) by leaving comments in the review thread below. (For completists: if the target issue tracker is also on GitHub, linking the review thread in the issue or vice versa will create corresponding breadcrumb trails in the link target.)

## Reviewer instructions & questions

@nmstreethran & @ibarraespinosa, please carry out your review in this issue by updating the checklist below. If you cannot edit the checklist please:
1. Make sure you're logged in to your GitHub account
2. Be sure to accept the invite at this URL: https://github.com/openjournals/joss-reviews/invitations

The reviewer guidelines are available here: https://joss.readthedocs.io/en/latest/reviewer_guidelines.html. Any questions/concerns please let @ajstewartlang know.

✨ **Please start on your review when you are able, and be sure to complete your review in the next six weeks, at the very latest** ✨ 

### Conflict of interest

- [x] I confirm that I have read the [JOSS conflict of interest (COI) policy](https://github.com/openjournals/joss/blob/master/COI.md) and that: I have no COIs with reviewing this work or that any perceived COIs have been waived by JOSS for the purpose of this review.

### Code of Conduct

- [x] I confirm that I read and will adhere to the [JOSS code of conduct](https://joss.theoj.org/about#code_of_conduct).

### General checks

- [x] **Repository:** Is the source code for this software available at the <a target="_blank" href="https://github.com/adithirgis/pollucheck">repository url</a>?
- [x] **License:** Does the repository contain a plain-text LICENSE file with the contents of an [OSI approved](https://opensource.org/licenses/alphabetical) software license?
- [x] **Contribution and authorship:** Has the submitting author (@adithirugis) made major contributions to the software? Does the full list of paper authors seem appropriate and complete?
- [x] **Substantial scholarly effort:** Does this submission meet the scope eligibility described in the [JOSS guidelines](https://joss.readthedocs.io/en/latest/submitting.html#substantial-scholarly-effort)

### Functionality

- [x] **Installation:** Does installation proceed as outlined in the documentation?
- [x] **Functionality:** Have the functional claims of the software been confirmed?
- [x] **Performance:** If there are any performance claims of the software, have they been confirmed? (If there are no claims, please check off this item.)

### Documentation

- [x] **A statement of need:** Do the authors clearly state what problems the software is designed to solve and who the target audience is?
- [x] **Installation instructions:** Is there a clearly-stated list of dependencies? Ideally these should be handled with an automated package management solution.
- [x] **Example usage:** Do the authors include examples of how to use the software (ideally to solve real-world analysis problems).
- [x] **Functionality documentation:** Is the core functionality of the software documented to a satisfactory level (e.g., API method documentation)?
- [x] **Automated tests:** Are there automated tests or manual steps described so that the functionality of the software can be verified?
- [x] **Community guidelines:** Are there clear guidelines for third parties wishing to 1) Contribute to the software 2) Report issues or problems with the software 3) Seek support

### Software paper

- [x] **Summary:** Has a clear description of the high-level functionality and purpose of the software for a diverse, non-specialist audience been provided?
- [x] **A statement of need:** Does the paper have a section titled 'Statement of Need' that clearly states what problems the software is designed to solve and who the target audience is?
- [x] **State of the field:** Do the authors describe how this software compares to other commonly-used packages?
- [x] **Quality of writing:** Is the paper well written (i.e., it does not require editing for structure, language, or writing quality)?
- [x] **References:** Is the list of references complete, and is everything cited appropriately that should be cited (e.g., papers, datasets, software)? Do references in the text use the proper [citation syntax]( https://rmarkdown.rstudio.com/authoring_bibliographies_and_citations.html#citation_syntax)?

### Conflict of interest

- [x] I confirm that I have read the [JOSS conflict of interest (COI) policy](https://github.com/openjournals/joss/blob/master/COI.md) and that: I have no COIs with reviewing this work or that any perceived COIs have been waived by JOSS for the purpose of this review.

### Code of Conduct

- [x] I confirm that I read and will adhere to the [JOSS code of conduct](https://joss.theoj.org/about#code_of_conduct).

### General checks

- [x] **Repository:** Is the source code for this software available at the <a target="_blank" href="https://github.com/adithirgis/pollucheck">repository url</a>?
- [x] **License:** Does the repository contain a plain-text LICENSE file with the contents of an [OSI approved](https://opensource.org/licenses/alphabetical) software license?
- [x] **Contribution and authorship:** Has the submitting author (@adithirugis) made major contributions to the software? Does the full list of paper authors seem appropriate and complete?
- [x] **Substantial scholarly effort:** Does this submission meet the scope eligibility described in the [JOSS guidelines](https://joss.readthedocs.io/en/latest/submitting.html#substantial-scholarly-effort)

### Functionality

- [x] **Installation:** Does installation proceed as outlined in the documentation?
- [x] **Functionality:** Have the functional claims of the software been confirmed?
- [x] **Performance:** If there are any performance claims of the software, have they been confirmed? (If there are no claims, please check off this item.)

### Documentation

- [x] **A statement of need:** Do the authors clearly state what problems the software is designed to solve and who the target audience is?
- [x] **Installation instructions:** Is there a clearly-stated list of dependencies? Ideally these should be handled with an automated package management solution.
- [x] **Example usage:** Do the authors include examples of how to use the software (ideally to solve real-world analysis problems).
- [x] **Functionality documentation:** Is the core functionality of the software documented to a satisfactory level (e.g., API method documentation)?
- [x] **Automated tests:** Are there automated tests or manual steps described so that the functionality of the software can be verified?
- [x] **Community guidelines:** Are there clear guidelines for third parties wishing to 1) Contribute to the software 2) Report issues or problems with the software 3) Seek support

### Software paper

- [x] **Summary:** Has a clear description of the high-level functionality and purpose of the software for a diverse, non-specialist audience been provided?
- [x] **A statement of need:** Does the paper have a section titled 'Statement of Need' that clearly states what problems the software is designed to solve and who the target audience is?
- [x] **State of the field:** Do the authors describe how this software compares to other commonly-used packages?
- [x] **Quality of writing:** Is the paper well written (i.e., it does not require editing for structure, language, or writing quality)?
- [x] **References:** Is the list of references complete, and is everything cited appropriately that should be cited (e.g., papers, datasets, software)? Do references in the text use the proper [citation syntax]( https://rmarkdown.rstudio.com/authoring_bibliographies_and_citations.html#citation_syntax)?

**Comment by whedon** [comment]
Failed to discover a `Statement of need` section in paper

**Comment by whedon** [comment]
```
Software report (experimental):

github.com/AlDanial/cloc v 1.88  T=0.20 s (139.7 files/s, 483198.0 lines/s)
-------------------------------------------------------------------------------
Language                     files          blank        comment           code
-------------------------------------------------------------------------------
JSON                            13            211              0          93166
R                                7             49             58           2251
Markdown                         6            265              0            590
TeX                              1              6              0             56
Rmd                              1             64            127              1
-------------------------------------------------------------------------------
SUM:                            28            595            185          96064
-------------------------------------------------------------------------------


Statistical information for the repository 'adbaf3aa7f52be1d2f44b18c' was
gathered on 2021/06/30.
No commited files with the specified extensions were found.
```

**Comment by whedon** [comment]
:point_right::page_facing_up: [Download article proof](https://raw.githubusercontent.com/openjournals/joss-papers/joss.03435/joss.03435/10.21105.joss.03435.pdf) :page_facing_up: [View article proof on GitHub](https://github.com/openjournals/joss-papers/blob/joss.03435/joss.03435/10.21105.joss.03435.pdf) :page_facing_up: :point_left:

**Comment by ibarraespinosa** [comment]
hello @ajstewartlang, I can't update the checklist.

**Comment by adithirgis** [comment]
@whedon generate pdf

**Comment by whedon** [comment]
:point_right::page_facing_up: [Download article proof](https://raw.githubusercontent.com/openjournals/joss-papers/joss.03435/joss.03435/10.21105.joss.03435.pdf) :page_facing_up: [View article proof on GitHub](https://github.com/openjournals/joss-papers/blob/joss.03435/joss.03435/10.21105.joss.03435.pdf) :page_facing_up: :point_left:

**Comment by adithirgis** [comment]
@whedon commands

**Comment by whedon** [comment]
Here are some things you can ask me to do:

```
# List Whedon's capabilities
@whedon commands

# List of editor GitHub usernames
@whedon list editors

# List of reviewers together with programming language preferences and domain expertise
@whedon list reviewers

EDITORIAL TASKS

# Compile the paper
@whedon generate pdf

# Compile the paper from alternative branch
@whedon generate pdf from branch custom-branch-name

# Ask Whedon to check the references for missing DOIs
@whedon check references

# Ask Whedon to check repository statistics for the submitted software
@whedon check repository

```

**Comment by adithirgis** [comment]
@whedon check references

**Comment by whedon** [comment]
```
Reference check summary (note 'MISSING' DOIs are suggestions that need verification):

OK DOIs

- 10.1093/cvr/cvaa092 is OK
- 10.18637/jss.v027.i03 is OK
- 10.1016/j.envsoft.2011.09.008 is OK
- 10.21105/joss.00333 is OK
- 10.21105/joss.02250 is OK

MISSING DOIs

- None

INVALID DOIs

- None
```

**Comment by ajstewartlang** [comment]
> hello @ajstewartlang, I can't update the checklist.

hi @ibarraespinosa can you double check that you've accepted the GH email invitation to this repo ok?

**Comment by ibarraespinosa** [comment]
I got this message
Sorry, we couldn't find that repository invitation. It is possible that the invitation was revoked or that you are not logged into the invited account.

**Comment by danielskatz** [comment]
@whedon re-invite @ibarraespinosa as reviewer

**Comment by whedon** [comment]
OK, the reviewer has been re-invited.

@ibarraespinosa please accept the invite by clicking this link: https://github.com/openjournals/joss-reviews/invitations

**Comment by ibarraespinosa** [comment]
Thanks 😊

Em qui, 8 de jul de 2021 09:58, whedon ***@***.***> escreveu:

**Comment by whedon** [comment]
:wave: @nmstreethran, please update us on how your review is going (this is an automated reminder).

**Comment by whedon** [comment]
:wave: @ibarraespinosa, please update us on how your review is going (this is an automated reminder).

**Comment by adithirgis** [comment]
@whedon check references

**Comment by whedon** [comment]
```
Reference check summary (note 'MISSING' DOIs are suggestions that need verification):

OK DOIs

- 10.1093/cvr/cvaa092 is OK
- 10.18637/jss.v027.i03 is OK
- 10.1016/j.envsoft.2011.09.008 is OK
- 10.21105/joss.00333 is OK
- 10.21105/joss.02250 is OK

MISSING DOIs

- None

INVALID DOIs

- None
```

**Comment by adithirgis** [comment]
@whedon generate pdf

**Comment by whedon** [comment]
:point_right::page_facing_up: [Download article proof](https://raw.githubusercontent.com/openjournals/joss-papers/joss.03435/joss.03435/10.21105.joss.03435.pdf) :page_facing_up: [View article proof on GitHub](https://github.com/openjournals/joss-papers/blob/joss.03435/joss.03435/10.21105.joss.03435.pdf) :page_facing_up: :point_left:

**Comment by adithirgis** [comment]
@whedon check references

**Comment by whedon** [comment]
```
Reference check summary (note 'MISSING' DOIs are suggestions that need verification):

OK DOIs

- 10.1093/cvr/cvaa092 is OK
- 10.18637/jss.v027.i03 is OK
- 10.1016/j.envsoft.2011.09.008 is OK
- 10.21105/joss.00333 is OK
- 10.21105/joss.02250 is OK
- 10.1016/j.asr.2021.02.008 is OK
- 10.1016/j.atmosenv.2007.09.003 is OK
- 10.1016/j.atmosenv.2014.11.062 is OK

MISSING DOIs

- None

INVALID DOIs

- None
```

**Comment by adithirgis** [comment]
@whedon generate pdf

**Comment by whedon** [comment]
:point_right::page_facing_up: [Download article proof](https://raw.githubusercontent.com/openjournals/joss-papers/joss.03435/joss.03435/10.21105.joss.03435.pdf) :page_facing_up: [View article proof on GitHub](https://github.com/openjournals/joss-papers/blob/joss.03435/joss.03435/10.21105.joss.03435.pdf) :page_facing_up: :point_left:

**Comment by adithirgis** [comment]
@whedon generate pdf

**Comment by whedon** [comment]
:point_right::page_facing_up: [Download article proof](https://raw.githubusercontent.com/openjournals/joss-papers/joss.03435/joss.03435/10.21105.joss.03435.pdf) :page_facing_up: [View article proof on GitHub](https://github.com/openjournals/joss-papers/blob/joss.03435/joss.03435/10.21105.joss.03435.pdf) :page_facing_up: :point_left:

**Comment by ibarraespinosa** [comment]
Hello  @ajstewartlang, I have already reviewed the paper by @adithirgis. It consists in a software to analyze air pollution measurements with different databases. **pollucheck** analyze different type of data-sets. I have been working with air pollution since 14 years ago in many countries and I`m sure this software will be used in many countries. Thanks for the invitation to review this paper and have a nice day.

**Comment by adithirgis** [comment]
Thank you so much @ibarraespinosa , we have addressed the OpenAQ issue, please let us know if you still face same issue.

**Comment by ajstewartlang** [comment]
Many thanks for your helpful review @ibarraespinosa - @nmstreethran are you getting on ok with your review?

**Comment by nmstreethran** [comment]
Hi @ajstewartlang, I'm working on the review just now and will open an issue in the target repository. I'll update you once my review is complete.

Thanks.

**Comment by adithirgis** [comment]
@whedon check references

**Comment by whedon** [comment]
```
Reference check summary (note 'MISSING' DOIs are suggestions that need verification):

OK DOIs

- 10.1093/cvr/cvaa092 is OK
- 10.18637/jss.v027.i03 is OK
- 10.1016/j.envsoft.2011.09.008 is OK
- 10.21105/joss.00333 is OK
- 10.21105/joss.02250 is OK
- 10.1016/j.asr.2021.02.008 is OK
- 10.1016/j.atmosenv.2007.09.003 is OK
- 10.1016/j.atmosenv.2014.11.062 is OK

MISSING DOIs

- None

INVALID DOIs

- None
```

**Comment by adithirgis** [comment]
@whedon generate pdf

**Comment by whedon** [comment]
:point_right::page_facing_up: [Download article proof](https://raw.githubusercontent.com/openjournals/joss-papers/joss.03435/joss.03435/10.21105.joss.03435.pdf) :page_facing_up: [View article proof on GitHub](https://github.com/openjournals/joss-papers/blob/joss.03435/joss.03435/10.21105.joss.03435.pdf) :page_facing_up: :point_left:

**Comment by adithirgis** [comment]
Hi @ibarraespinosa!

We have implemented your suggestion of adding case study. 

Thank you!

**Comment by ibarraespinosa** [comment]
Hi Adithi, thanks you!.

Em ter., 20 de jul. de 2021 às 08:31, Adithi R. Upadhya <
***@***.***> escreveu:

**Comment by adithirgis** [comment]
@whedon generate pdf

**Comment by whedon** [comment]
:point_right::page_facing_up: [Download article proof](https://raw.githubusercontent.com/openjournals/joss-papers/joss.03435/joss.03435/10.21105.joss.03435.pdf) :page_facing_up: [View article proof on GitHub](https://github.com/openjournals/joss-papers/blob/joss.03435/joss.03435/10.21105.joss.03435.pdf) :page_facing_up: :point_left:

**Comment by adithirgis** [comment]
@whedon generate pdf

**Comment by whedon** [comment]
:point_right::page_facing_up: [Download article proof](https://raw.githubusercontent.com/openjournals/joss-papers/joss.03435/joss.03435/10.21105.joss.03435.pdf) :page_facing_up: [View article proof on GitHub](https://github.com/openjournals/joss-papers/blob/joss.03435/joss.03435/10.21105.joss.03435.pdf) :page_facing_up: :point_left:

**Comment by adithirgis** [comment]
@whedon generate pdf

**Comment by adithirgis** [comment]
@whedon generate pdf

**Comment by whedon** [comment]
:point_right::page_facing_up: [Download article proof](https://raw.githubusercontent.com/openjournals/joss-papers/joss.03435/joss.03435/10.21105.joss.03435.pdf) :page_facing_up: [View article proof on GitHub](https://github.com/openjournals/joss-papers/blob/joss.03435/joss.03435/10.21105.joss.03435.pdf) :page_facing_up: :point_left:

**Comment by whedon** [comment]
:point_right::page_facing_up: [Download article proof](https://raw.githubusercontent.com/openjournals/joss-papers/joss.03435/joss.03435/10.21105.joss.03435.pdf) :page_facing_up: [View article proof on GitHub](https://github.com/openjournals/joss-papers/blob/joss.03435/joss.03435/10.21105.joss.03435.pdf) :page_facing_up: :point_left:

**Comment by nmstreethran** [comment]
Hi @ajstewartlang, I have completed my review (adithirgis/pollucheck#2). `pollucheck` is very useful for performing exploratory data analysis on open air pollution data. The app provides users an interactive way to do this without needing any programming knowledge. I'm happy to recommend this submission for publication. Please let me know if there is anything else I can do.

Thanks!

**Comment by adithirgis** [comment]
Thanks @ibarraespinosa and @nmstreethran for a thorough review. 

Thanks @ajstewartlang!

**Comment by ajstewartlang** [comment]
@whedon generate pdf

**Comment by whedon** [comment]
:point_right::page_facing_up: [Download article proof](https://raw.githubusercontent.com/openjournals/joss-papers/joss.03435/joss.03435/10.21105.joss.03435.pdf) :page_facing_up: [View article proof on GitHub](https://github.com/openjournals/joss-papers/blob/joss.03435/joss.03435/10.21105.joss.03435.pdf) :page_facing_up: :point_left:

**Comment by ajstewartlang** [comment]
@whedon check references

**Comment by whedon** [comment]
```
Reference check summary (note 'MISSING' DOIs are suggestions that need verification):

OK DOIs

- 10.1093/cvr/cvaa092 is OK
- 10.18637/jss.v027.i03 is OK
- 10.1016/j.envsoft.2011.09.008 is OK
- 10.21105/joss.00333 is OK
- 10.21105/joss.02250 is OK
- 10.1016/j.asr.2021.02.008 is OK
- 10.1016/j.atmosenv.2007.09.003 is OK
- 10.1016/j.atmosenv.2014.11.062 is OK

MISSING DOIs

- None

INVALID DOIs

- None
```

**Comment by ajstewartlang** [comment]
Many thanks @ibarraespinosa and @nmstreethran for your helpful and detailed reviews. I've had a read through myself @adithirgis and agree that `pollucheck` is a very nice and useful piece of software. Congrats!

@adithirgis can I now ask you to do the following please?

- Make a [tagged release](https://docs.github.com/en/github/administering-a-repository/releasing-projects-on-github/managing-releases-in-a-repository) of the software.
- Archive the reviewed software in Zenodo or a similar service (e.g., figshare, an institutional repository)
- Check the archival deposit (e.g., in Zenodo) has the correct metadata. This includes the title (should match the paper title) and author list (make sure the list is correct and people who only made a small fix are not on it). You may also add the authors' ORCID.
- Please list the DOI of the archived version here.

**Comment by adithirgis** [comment]
Thanks @ajstewartlang!
 
 - We made a tagged release of the v1.0.0
 - Here is the link to the metadata - https://zenodo.org/record/5126939#.YPqqUegzZPY (I checked the deposit)
 - DOI version from Zenodo - 10.5281/zenodo.5126939.
 
 Let me know if there is any issue with these. 
 
 Thanks again!
 Adithi

**Comment by ajstewartlang** [comment]
@whedon set 10.5281/zenodo.5126939 as archive

**Comment by whedon** [comment]
OK. <a href="https://doi.org/10.5281/zenodo.5126939" target="_blank">10.5281/zenodo.5126939</a> is the archive.

**Comment by ajstewartlang** [comment]
@whedon recommend-accept

**Comment by whedon** [comment]
```
Attempting dry run of processing paper acceptance...
```

**Comment by whedon** [comment]
```
Reference check summary (note 'MISSING' DOIs are suggestions that need verification):

OK DOIs

- 10.1093/cvr/cvaa092 is OK
- 10.18637/jss.v027.i03 is OK
- 10.1016/j.envsoft.2011.09.008 is OK
- 10.21105/joss.00333 is OK
- 10.21105/joss.02250 is OK
- 10.1016/j.asr.2021.02.008 is OK
- 10.1016/j.atmosenv.2007.09.003 is OK
- 10.1016/j.atmosenv.2014.11.062 is OK

MISSING DOIs

- None

INVALID DOIs

- None
```

**Comment by whedon** [comment]
:wave: @openjournals/joss-eics, this paper is ready to be accepted and published.

 Check final proof :point_right: https://github.com/openjournals/joss-papers/pull/2475

If the paper PDF and Crossref deposit XML look good in https://github.com/openjournals/joss-papers/pull/2475, then you can now move forward with accepting the submission by compiling again with the flag `deposit=true` e.g.
 ```
@whedon accept deposit=true
```

**Comment by danielskatz** [comment]
👋 @adithirgis  - I've suggested a few small changes to the paper in https://github.com/adithirgis/pollucheck/pull/3 - please merge this, or let me know what you disagree with, then we can proceed to final acceptance and publishing.

**Comment by danielskatz** [comment]
@whedon recommend-accept

**Comment by whedon** [comment]
```
Attempting dry run of processing paper acceptance...
```

**Comment by whedon** [comment]
```
Reference check summary (note 'MISSING' DOIs are suggestions that need verification):

OK DOIs

- 10.1093/cvr/cvaa092 is OK
- 10.18637/jss.v027.i03 is OK
- 10.1016/j.envsoft.2011.09.008 is OK
- 10.21105/joss.00333 is OK
- 10.21105/joss.02250 is OK
- 10.1016/j.asr.2021.02.008 is OK
- 10.1016/j.atmosenv.2007.09.003 is OK
- 10.1016/j.atmosenv.2014.11.062 is OK

MISSING DOIs

- None

INVALID DOIs

- None
```

**Comment by whedon** [comment]
:wave: @openjournals/joss-eics, this paper is ready to be accepted and published.

 Check final proof :point_right: https://github.com/openjournals/joss-papers/pull/2476

If the paper PDF and Crossref deposit XML look good in https://github.com/openjournals/joss-papers/pull/2476, then you can now move forward with accepting the submission by compiling again with the flag `deposit=true` e.g.
 ```
@whedon accept deposit=true
```

**Comment by adithirgis** [comment]
Hi @danielskatz,

Is it possible to change the Zenodo now?

**Comment by danielskatz** [comment]
you don't need to, as we don't need the zenodo archive to have the paper at all, let alone the final version, but if you want to, that's fine - you could update it.  Let me know if you want to to, and the new DOI (and new version tag) if so

**Comment by adithirgis** [comment]
Hi @danielskatz,

Here is the new DOI - 10.5281/zenodo.5128607, no new version tag
Thanks again!

**Comment by danielskatz** [comment]
@whedon set 10.5281/zenodo.5128607 as archive

**Comment by whedon** [comment]
OK. <a href="https://doi.org/10.5281/zenodo.5128607" target="_blank">10.5281/zenodo.5128607</a> is the archive.

**Comment by danielskatz** [comment]
@whedon recommend-accept

**Comment by whedon** [comment]
```
Attempting dry run of processing paper acceptance...
```

**Comment by whedon** [comment]
```
Reference check summary (note 'MISSING' DOIs are suggestions that need verification):

OK DOIs

- 10.1093/cvr/cvaa092 is OK
- 10.18637/jss.v027.i03 is OK
- 10.1016/j.envsoft.2011.09.008 is OK
- 10.21105/joss.00333 is OK
- 10.21105/joss.02250 is OK
- 10.1016/j.asr.2021.02.008 is OK
- 10.1016/j.atmosenv.2007.09.003 is OK
- 10.1016/j.atmosenv.2014.11.062 is OK

MISSING DOIs

- None

INVALID DOIs

- None
```

**Comment by whedon** [comment]
:wave: @openjournals/joss-eics, this paper is ready to be accepted and published.

 Check final proof :point_right: https://github.com/openjournals/joss-papers/pull/2477

If the paper PDF and Crossref deposit XML look good in https://github.com/openjournals/joss-papers/pull/2477, then you can now move forward with accepting the submission by compiling again with the flag `deposit=true` e.g.
 ```
@whedon accept deposit=true
```

**Comment by danielskatz** [comment]
@whedon accept deposit=true

**Comment by whedon** [comment]
```
Doing it live! Attempting automated processing of paper acceptance...
```

**Comment by whedon** [comment]
🐦🐦🐦 👉 [Tweet for this paper](https://twitter.com/JOSS_TheOJ/status/1418584612356378632) 👈 🐦🐦🐦

**Comment by whedon** [comment]
🚨🚨🚨 **THIS IS NOT A DRILL, YOU HAVE JUST ACCEPTED A PAPER INTO JOSS!** 🚨🚨🚨

 Here's what you must now do:

0. Check final PDF and Crossref metadata that was deposited :point_right: https://github.com/openjournals/joss-papers/pull/2478
1. Wait a couple of minutes, then verify that the paper DOI resolves [https://doi.org/10.21105/joss.03435](https://doi.org/10.21105/joss.03435)
2. If everything looks good, then close this review issue.
3. Party like you just published a paper! 🎉🌈🦄💃👻🤘

 Any issues? Notify your editorial technical team...

**Comment by adithirgis** [comment]
> Any issues? Notify your editorial technical team...

I just updated a little, will that be also highlighted?

**Comment by danielskatz** [comment]
I'm not sure what you mean by "highlighted".  When I did the accept, it captured what was in the paper.md and paper.bib at that time.  Did you make changes after that?

**Comment by adithirgis** [comment]
Oh ok, no worries. 

Thanks! I did but I think we can skip that. 

Sorry for the inconvenience caused.

**Comment by danielskatz** [comment]
Congratulations to @adithirgis (Adithi Upadhya) and co-authors!!

And thanks to @nmstreethran and @ibarraespinosa for reviewing, and @ajstewartlang for editing!

**Comment by whedon** [comment]
:tada::tada::tada: Congratulations on your paper acceptance! :tada::tada::tada:

If you would like to include a link to your paper from your README use the following code snippets:

```
Markdown:
[![DOI](https://joss.theoj.org/papers/10.21105/joss.03435/status.svg)](https://doi.org/10.21105/joss.03435)

HTML:
<a style="border-width:0" href="https://doi.org/10.21105/joss.03435">
  <img src="https://joss.theoj.org/papers/10.21105/joss.03435/status.svg" alt="DOI badge" >
</a>

reStructuredText:
.. image:: https://joss.theoj.org/papers/10.21105/joss.03435/status.svg
   :target: https://doi.org/10.21105/joss.03435
```
This is how it will look in your documentation:

[![DOI](https://joss.theoj.org/papers/10.21105/joss.03435/status.svg)](https://doi.org/10.21105/joss.03435)

**We need your help!**

Journal of Open Source Software is a community-run journal and relies upon volunteer effort. If you'd like to support us please consider doing either one (or both) of the the following:

- Volunteering to review for us sometime in the future. You can add your name to the reviewer list here: https://joss.theoj.org/reviewer-signup.html
- Making a small donation to support our running costs here: https://numfocus.org/donate-to-joss
